test/Microsoft.Framework.PackageManager.Tests/project.json

```diff
@@ -2,7 +2,6 @@
     "dependencies": {
         "Microsoft.Framework.CommonTestUtils": "1.0.0-*",
         "Microsoft.Framework.PackageManager": "1.0.0-*",
-        "Microsoft.Framework.Runtime.Sources": "1.0.0-*",
```

Comment:
I'm not sure why my change revealed this but this was conflicting with the versions used in Microsoft.Framework.PackageManager which were visible because that has InternalsVisibleTo this assembly